substrate/frame/staking/src/slashing.rs

```diff
@@ -280,18 +278,13 @@ pub(crate) fn compute_slash<T: Config>(
 		let target_span = spans.compare_and_update_span_slash(params.slash_era, own_slash);
 
 		if target_span == Some(spans.span_index()) {
-			// misbehavior occurred within the current slashing span - take appropriate
-			// actions.
-
-			// chill the validator - it misbehaved in the current span and should
-			// not continue in the next election. also end the slashing span.
+			// misbehavior occurred within the current slashing span - end current span.
+			// Check <https://github.com/paritytech/polkadot-sdk/issues/2650> for details.
 			spans.end_span(params.now);
-			<Pallet<T>>::chill_stash(params.stash);
```

Comment:
Cross posting question on chilling here
https://github.com/paritytech/polkadot-sdk/pull/2955/files#r1519626378

Reply:
From the linked comment:

> Disabling nodes would not protect nominators as the validator would get re-elected in the next era and may get slashed for the same offence again. Chilling works similarly but its up to validator to signal that they have fixed the issue that got them slashed and ready to be considered for re-election.

Chilling protects nominators and honest validators from "honest mistakes". By honest here I mean something not malicious but still not correct. E.g. misconfigured validator or something similar.

Disabling is designed to protect the network and the nominators from malicious validators. A dishonest node can always unchill itself and continue do damage (and get slashed). This is not possible with disabling.